proposals/simd/NewOpcodes.md

```diff
@@ -80,13 +80,13 @@
 
 | i8x16 Op             | opcode | i16x8 Op                 | opcode | i32x4 Op                 | opcode | i64x2 Op                 | opcode |
 | -------------------- | ------ | ------------------------ | ------ | ------------------------ | ------ | ------------------------ | ------ |
-| i8x16.abs            | 0x60   | i16x8.abs                | 0x80   | i32x4.abs                | 0xa0   | ----                     | 0xc0   |
+| i8x16.abs            | 0x60   | i16x8.abs                | 0x80   | i32x4.abs                | 0xa0   | -------------            | 0xc0   |
 | i8x16.neg            | 0x61   | i16x8.neg                | 0x81   | i32x4.neg                | 0xa1   | i64x2.neg                | 0xc1   |
-| i8x16.any_true       | 0x62   | i16x8.any_true           | 0x82   | i32x4.any_true           | 0xa2   | ----                     | 0xc2   |
-| i8x16.all_true       | 0x63   | i16x8.all_true           | 0x83   | i32x4.all_true           | 0xa3   | ----                     | 0xc3   |
+| -------------        | 0x62   | -------------            | 0x82   | -------------            | 0xa2   | -------------            | 0xc2   |
+| i8x16.all_true       | 0x63   | i16x8.all_true           | 0x83   | i32x4.all_true           | 0xa3   | -------------            | 0xc3   |
```

Comment:
@Maratyszcza Hi, I understand the removal of the type specific any_true but should the v128.any_true be associated with an opcode somewhere in this file?

Reply:
The opcodes is TBD. IIUC, the plan is to assign opcodes for newly added instructions once the instruction set is finalized.

Reply:
#421 tracks TBD opcodes.